constants.php

```diff
@@ -6,7 +6,7 @@
 {
 	define(__NAMESPACE__ . '\running', true);
 	define(__NAMESPACE__ . '\directory', __DIR__);
-	define(__NAMESPACE__ . '\version', preg_replace('/\$Rev: ([^ ]+) \$/', '$1', '$Rev: DEVELOPMENT-2.1.0 $'));
+	define(__NAMESPACE__ . '\version', preg_replace('/\$Rev: ([^ ]+) \$/', '$1', '$Rev: DEVELOPMENT-2.2.0 $'));
```

Comment:
as @mageekguy spotted some time ago, I think we are doing something wrong here : we are releasing the `2.2.0` and we mark it as `DEVELOPMENT` which is wrong.

I think we should remove the ` DEVELOPMENT` word here and as soon as the tag is pushed, we bump to the next version : `DEVELOPMENT-2.3.0`